grakn-graql-shell/src/main/java/ai/grakn/graql/GraqlShell.java

```diff
@@ -222,10 +208,17 @@ public static boolean runShell(String[] args, String version, String historyFile
         }
 
         Keyspace keyspace = Keyspace.of(cmd.getOptionValue("k", DEFAULT_KEYSPACE));
-        SimpleURI location = Optional.ofNullable(cmd.getOptionValue("r")).map(SimpleURI::new).orElse(Grakn.DEFAULT_URI);
+
+        int defaultGrpcPort = 48555;
```

Comment:
Should we this value it from config file? Maybe not now, but we will probably need a `client-grakn.properties` at some point so that we can expose only a subset of settings to the client's user.

Reply:
Maybe we can get it from the config file! I'll try it and see.